### PR TITLE
Add project overview and list of repositories

### PR DESCRIPTION
**Pull request by sirkrypt0**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# Hey 👋
+# Using FIDO2 In Electronic Locking Systems 🔐
+
+![Unlocking an electronic lock using a FIDO2 hardware authenticator](img/unlock.jpeg)
+
+This group is a result of the *Hot Topics in Secure Identity Research* seminar at the Hasso-Plattner-Institute (HPI).
```
Comment (felix-gohla) [inline profile/README.md]: Link to HPI?

File: profile/README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+In one semester, we evaluated whether FIDO2 can be used not only for authentication in the web,
+but also for authentication offline in electronic locking systems.
+
+This organization holds the changes we made to existing tools in order to build a working proof
+of concept.
+It uses a Solo 2 hacker edition as the FIDO2 authenticator, an ACR-122U NFC reader, a Raspberry Pi 3B+, and some status LEDs.
```
Comment (felix-gohla) [inline profile/README.md]: Link to solo keys or kickstarter?

File: profile/README.md [hunk 3]
```diff
@@ -1 +1,41 @@
...
+- [ctaphid-dispatch](https://github.com/All-Your-Locks-Are-Belong-To-Us/ctaphid-dispatch)
+- [lpc55-host](https://github.com/All-Your-Locks-Are-Belong-To-Us/lpc55-host/tree/fix/configure-pfr)
+
+To test our firmware changes and simulate a lock, we developed a small suite of tools for FIDO2.
+As Chromium uses GZIP we added the GZIP compression to the `libfido2`.
+
+- [fido2-debug-client](https://github.com/All-Your-Locks-Are-Belong-To-Us/fido2-debug-client)
+- [libfido2](https://github.com/All-Your-Locks-Are-Belong-To-Us/libfido2/tree/feature/large-blob-use-gzip)
+
+Finally, we write access rights onto a FIDO2 authenticator using a small web application.
```
Comment (felix-gohla) [inline profile/README.md:36]: Link to wau?

**Comment by felix-gohla** [comment]
It's a little redundant to the docs in Gitlab, isn't it? 🤔

File: profile/README.md [hunk 4]
```diff
@@ -1 +1,41 @@
...
+- [fido-authenticator](https://github.com/All-Your-Locks-Are-Belong-To-Us/fido-authenticator/tree/feature/fido-2-1-large-blobs)
+- [ctap-types](https://github.com/All-Your-Locks-Are-Belong-To-Us/ctap-types/tree/feature/fido-2-1-largeBlobs)
+- [apdu-dispatch](https://github.com/All-Your-Locks-Are-Belong-To-Us/apdu-dispatch)
+- [oath-authenticator](https://github.com/All-Your-Locks-Are-Belong-To-Us/oath-authenticator)
+- [admin-app](https://github.com/All-Your-Locks-Are-Belong-To-Us/admin-app)
+- [ctaphid-dispatch](https://github.com/All-Your-Locks-Are-Belong-To-Us/ctaphid-dispatch)
+- [lpc55-host](https://github.com/All-Your-Locks-Are-Belong-To-Us/lpc55-host/tree/fix/configure-pfr)
+
+To test our firmware changes and simulate a lock, we developed a small suite of tools for FIDO2.
+As Chromium uses GZIP we added the GZIP compression to the `libfido2`.
```
Comment (konradh) [inline profile/README.md]:
```suggestion
Even though the CTAP2.1 standard specifies DEFLATE (RFC 1951) for large blob compression, Chromium uses GZIP (RFC 1952). Therefore, we added GZIP compression to the `libfido2` for now. We will file a bug for Chromium to fix this behavior.
```

**Review by felix-gohla — CHANGES_REQUESTED** [review]

**Review by j-hellenberg — APPROVED** [review]

**Review by felix-gohla — APPROVED** [review]
lgtm

**Pull request merged**